pkg/ccl/streamingccl/streamproducer/span_config_event_stream.go

```diff
@@ -217,7 +223,10 @@ func (s *spanConfigEventStream) streamLoop(ctx context.Context) error {
 		case update := <-s.updateCh:
 			for _, ev := range update.Events {
 				spcfgEvent := ev.(*spanconfigkvsubscriber.BufferEvent)
-				target := spcfgEvent.Update.GetTarget().ToProto()
+				target := spcfgEvent.Update.GetTarget()
+				if target.IsSystemTarget() {
+					continue
+				}
```

Comment:
Since we aren't experts in the span config stuff, I think some comments around what this means and why we are skipping it would be useful.

Reply:
Done.